### PR TITLE
Rewrite the config visualization page

### PR DESCRIPTION
**Pull request by fsimonis**
This PR updates the config-visualization page with the recent changes wrt the CLI and the GUI

### CODE REVIEW

File: pages/docs/tooling/tooling-config-visualization.md [hunk 1]
```diff
@@ -22,36 +22,37 @@ This has a few important benefits:
 
 ## Installation
 
+We provide two python packages ways of visualizing preCICE configurations.
+
+1. `precice-config-visualizer` which translate preCICE configuration files to a graph and optionally renders them as an image.
```
Comment (MakisH) [inline pages/docs/tooling/tooling-config-visualization.md]:
```suggestion
1. `precice-config-visualizer` which translates preCICE configuration files to a graph and optionally renders them as an image.
```

File: pages/docs/tooling/tooling-config-visualization.md [hunk 2]
```diff
@@ -22,36 +22,37 @@ This has a few important benefits:
...
 ```bash
 pipx install precice-config-visualizer
-```
-
-Alternatively, you can install the latest develop version from the repository
-
-```bash
-pipx install https://github.com/precice/config-visualizer/archive/master.zip
+# To also install the GUI
+pipx install 'precice-config-visualizer[gui]'
```
Comment (MakisH) [inline pages/docs/tooling/tooling-config-visualization.md:47]: Opened an issue too fast, but see https://github.com/precice/config-visualizer-gui/issues/1#issue-2238529152

**Review by MakisH — APPROVED** [review]

File: pages/docs/tooling/tooling-config-visualization.md (hunk 2 above)
Comment (MakisH) [inline pages/docs/tooling/tooling-config-visualization.md]:
```suggestion
pipx install 'precice-config-visualizer-gui'
```

Opened an issue too fast, but see https://github.com/precice/config-visualizer-gui/issues/1#issue-2238529152

File: pages/docs/tooling/tooling-config-visualization.md [hunk 3]
```diff
@@ -65,27 +66,43 @@ precice-config-visualizer-gui precice-config.xml
 
 The application automatically reloads configuration files on change and shows parsing errors at the bottom. This is especially useful for rapid prototyping.
 
-Desktop integration is still lacking with commonly used tools for installin python packages.
-If you want your launcher to pick up the tool, you can save [its desktop file](https://raw.githubusercontent.com/precice/config-visualizer/master/data/org.precice.config_visualizer.desktop) manually to the directory `~/.local/share/applications/`. The directory may need to be created first.
+Desktop integration is still lacking with commonly used tools for installing python packages.
+If you want your launcher to pick up the tool, you can save [this desktop file](https://gist.githubusercontent.com/fsimonis/a08c3771abf808b0534d658bcb563f90/raw/10717f5b2afbf2d3c05ecb9f5c7eeaae4bbe868a/org.precice.configvisualizer.desktop) (right-click > "Save Link As") manually to the directory `~/.local/share/applications/`. The directory may need to be created first.
```
Comment (MakisH) [inline pages/docs/tooling/tooling-config-visualization.md]: Did not try this yet, too many tabs and programs open over here at the moment.

Comment (fsimonis) [reply pages/docs/tooling/tooling-config-visualization.md]: _skill issue_ :wink:

File: pages/docs/tooling/tooling-config-visualization.md (hunk 2 above)
Comment (fsimonis) [reply pages/docs/tooling/tooling-config-visualization.md]: Actually, no ticks needed then.

```suggestion
pipx install precice-config-visualizer-gui
```

File: pages/docs/tooling/tooling-config-visualization.md (hunk 3 above)
Comment (MakisH) [reply pages/docs/tooling/tooling-config-visualization.md]: _you would know!_

(I had to consult [wikiHow](https://www.wikihow.com/What-Does-Skill-Issue-Mean) on understanding what you wrote me and how to respond)

Comment (MakisH) [reply pages/docs/tooling/tooling-config-visualization.md]: Somehow, it does not appear on my apps list. For comparison, here is the `webex.desktop` I have in the same directory (and which I see in my apps list):

```
[Desktop Entry]
Version=43.11.0.27795
Name=Webex
Comment=Webex
Exec=/opt/Webex/bin/CiscoCollabHost %U
Icon=/home/gc/.local/share/WebexLauncher/sparklogosmall.png
Terminal=false
Type=Application
Categories=Utility;Application;
MimeType=x-scheme-handler/webexteams;x-scheme-handler/ciscospark;x-scheme-handler/webex
```

Looks pretty similar to me.

Both files are marked as not executable.

Comment (fsimonis) [reply pages/docs/tooling/tooling-config-visualization.md]: Did you logout and login again? Sometimes a restart is necessary to pick this up too.

Comment (MakisH) [reply pages/docs/tooling/tooling-config-visualization.md]: yes, I restarted.

Comment (fsimonis) [reply pages/docs/tooling/tooling-config-visualization.md]: ~~Can you try if manually replacing `$HOME` works?~~

Tested it locally. The GNOME launcher doesn't expand HOME for some reason, when checking Exec entries in desktop files for correctness.

A workaround is to use a shell. I'll update the gist. @MakisH could you try again?

Comment (fsimonis) [reply pages/docs/tooling/tooling-config-visualization.md]:
```suggestion
If you want your launcher to pick up the tool, you can save [this desktop file](https://gist.githubusercontent.com/fsimonis/a08c3771abf808b0534d658bcb563f90/raw/e091d78c24b04d0fc903b8de4909528628d22b7b/org.precice.configvisualizer.desktop) (right-click > "Save Link As") manually to the directory `~/.local/share/applications/`. The directory may need to be created first.
```

Comment (MakisH) [reply pages/docs/tooling/tooling-config-visualization.md]: Yes, this worked (assuming it's the same you sent me privately).

**Review by MakisH — APPROVED** [review]

**Pull request merged**